src/abstract_register.jl

```diff
@@ -73,6 +73,19 @@ If only an integer is provided, then returns a lambda function.
 
 addbits!(n::Int) = @λ(register -> addbits!(register, n))
 
+"""
+    insert_qubits!(register, loc::Int; n::Int=1) -> register
```

Comment:
I basically understand why you are using a keyword for number of qubits, but this is not consistent with other interfaces in Base and ArrayReg. We usually use `n` as arg, not kwargs, I didn't find any interface using `n` as kwarg either. thus if you still want it to be keyword a longer name `nuqbits` or `total` would be better.

Reply:
agree

Reply:
I'm wondering if this is more consistent

```
insert_qubits!(register, n; loc)
```

since we use number of qubits as arguement behind register in other interface as well.

Reply:
`insert!(vector, loc, item)` is more consistent.